cinema/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=False)
```

Comment:
these fields are required in the model, but not required in serializer, that's strange